app/browser/movie-search.mjs

```diff
@@ -1,5 +1,75 @@
 /* globals document HTMLElement customElements */
 import formatTitle from '../lib/formatTitle.mjs'
+import MoviePosterHTML from '../elements/movie-poster.mjs'
+import StarRatingHTML from '../elements/star-rating.mjs'
+const api = {
```

Comment:
This isn't "fully integrated" so we can either remove the store refs or decide how we'd like to wire it up.